src/bin/ion/commands/beta/generate/generator.rs

```diff
@@ -38,7 +38,10 @@ impl<'a> CodeGenerator<'a, RustLanguage> {
         .unwrap();
 
         // Render the imports into output file
-        let rendered = tera.render("import.templ", &Context::new()).unwrap();
+        let rendered_import = tera.render("import.templ", &Context::new()).unwrap();
+        // Render the SerdeResult that is used in generated read-write APIs
+        let rendered_result = tera.render("result.templ", &Context::new()).unwrap();
```

Comment:
(🗺️ PR tour) This statement renders `SerdeResult` for Rust code generation.

Reply:
I think this might not be a good idea. If we need a `SerdeResult` of some sort, it should probably be defined in a "ion-codegen-runtime" crate.

Reply:
Copied from another comment for convenient reference:

> Per offline discussion, `SerdeResult` will be a type that is generated alongside all the other code. This will guarantee that it's always available in codebases using codegen and does not require packaging a library per target language. It also means that we can evolve it more easily over time.